lua/elixir/mix/exs.lua

```diff
@@ -0,0 +1,18 @@
+local git = require("elixir.mix.git")
+
+local M = {}
+
+function M.path_mix_exs()
```

Comment:
I did this to define if the current directory is a mix project but I think the solution you already have in your repo is better.

Reply:
Cool, I can re-use!